spec/formatting.md

```diff
@@ -284,9 +488,48 @@ when *   {Other match}
 
 4. The pattern `{Exact match}` of the most preferred `1` variant is selected.
 
-## Handling Bidirectional Text
+## Formatting
+
+After _pattern selection_,
+each _text_ and _expression_ of the selected _pattern_ must be resolved and formatted.
```

Comment:
```suggestion
each _text_ and _expression_ part of the selected _pattern_ must be resolved and formatted.
```

(Not sure if we have a standard name for the parts/elements/components of a pattern yet.)

Reply:
We don't; that's another possible addition to the syntax like `operand` that we might want to make for the sake of clarity.